.github/workflows/test.yml

```
@@ -23,7 +23,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    
+      with:
+        submodules: recursive
+    - name: Make OpenSurgery into a package
+        run: |
+          touch src/lsqecc/external/opensurgery/__init__.py
```

```suggestion
    - name: Make OpenSurgery into a package
      run: |
        touch src/lsqecc/external/opensurgery/__init__.py
```
Alignment issue caused the failure I think